Makefile

```diff
@@ -267,7 +267,7 @@ run-clojure: ##@run Watch for and build Clojure changes for mobile
 
 run-metro: export TARGET := clojure
 run-metro: ##@run Start Metro to build React Native changes
-	@scripts/start-react-native.sh
+	@scripts/start-metro.sh
```

Comment:
Probably should be `run-metro.sh`.

Reply:
addressed here : https://github.com/status-im/status-mobile/pull/18784/commits/d6a19d75da1010ab9e33dd395ba4c6e45e7456e6